packages/runtime/src/runtimes/react/components/page/index.tsx

```diff
@@ -30,26 +30,48 @@ function useRegisterPageDocument(pageDocument: MakeswiftPageDocument): ReactPage
   return rootDocuments[0]
 }
 
-export const Page = ({
-  snapshot,
-  metadata,
-}: {
-  snapshot: MakeswiftPageSnapshot
-  metadata: boolean | PageMetadataSettings
-}) => {
-  useCacheData(snapshot.cacheData)
-
-  const rootDocument = useRegisterPageDocument(snapshot.document)
-
-  return (
-    <Suspense>
-      {/* We use a key here to reset the Snippets state in the PageMeta component */}
-      <PageComponent
-        key={snapshot.document.data.key}
-        page={snapshot.document}
-        rootDocument={rootDocument}
-        metadata={metadata}
-      />
-    </Suspense>
-  )
-}
+/**
+ * @param snapshot - The snapshot of the page to render, from
+ * `client.getPageSnapshot()`.
+ * @param metadata - Allows control over whether to use data from Makeswift for
+ * rendering metadata tags in the `<head>` of the page. Pass `true` (default if
+ * not provided) to render all metadata tags, or `false` to not render any. For
+ * more granular control, pass an object with boolean values for specific
+ * metadata fields. Valid fields include:
+ *  - `title`
+ *  - `description`
+ *  - `keywords`
+ *  - `socialImage`
+ *  - `canonicalUrl`
+ *  - `indexingBlocked`
+ *  - `favicon`
+ *
+ * If a field is not provided, it will default to `false`.
+ */
```

Comment:
Document the decoupled built-in `Page` as well, since this is what's going to be directly exported in other frameworks.